pkg/reconciler/claimbinding/configurator.go

```diff
@@ -51,16 +52,16 @@ func NewObjectMetaConfigurator(_ runtime.ObjectTyper) *ObjectMetaConfigurator {
 }
 
 // Configure the supplied Managed resource's object metadata.
-func (c *ObjectMetaConfigurator) Configure(ctx context.Context, cm Claim, cs Class, mg Managed) error {
+func (c *ObjectMetaConfigurator) Configure(ctx context.Context, cm resource.Claim, cs resource.Class, mg resource.Managed) error {
 	return ConfigureNames(ctx, cm, cs, mg)
 }
 
 // ConfigureNames configures the name and external name of the supplied managed
-// resource. The managed resource name is derived from the supplied resource
+// resource. The managed resource name is derived from d resource
```

Comment:
I am thinking this is an unintentional change here.

Reply:
It was - thanks!